ProofOfAddress/README.md

```diff
@@ -56,7 +62,7 @@ On Linux, if your docker daemon runs with privileges, you may need to prefix `cd
 To build and deploy the sample, run the following commands:
 
 ```bash
-git clone https://github.com/aws-samples/i-love-my-local-farmer
+git clone https://gitlab.aws.dev/i-love-my-local-farmer/proof-of-address.git
```

Comment:
Are you sure this URL is correct? This is our internal GitLab instance and it is not publicly available.

Reply:
Fixed